Formula/pegtl.rb

```diff
@@ -15,19 +15,27 @@ class Pegtl < Formula
 
   depends_on "cmake" => :build
 
+  if MacOS.version <= :mojave
+    depends_on "gcc"
+    fails_with :clang do
+      cause "'path' is unavailable: introduced in macOS 10.15"
```

Comment:
The error message about path is not very clear to me, maybe add `C++17` somewhere so people understand better?

Reply:
I actually directly adopt the same change from somewhere. Can you make a code suggestion, I can adopt it.

Reply:
ping @fxcoudert :)

Reply:
Sorry, spending all my time trying to make CI nodes run again. It's a minor thing, do as you wish.